docs/_docs/best-practices.md

```diff
@@ -7,10 +7,10 @@ permalink: /docs/best-practices
 
 ## Coding Style
 
-#### Guidelines: ####
+#### Guidelines: ####=
```

Comment:
Whoops! seems I hit '=' instead of my Delete key. Sorry about that! Reverting now :D